subprojects/tree-sitter.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = tree-sitter-0.15.7
+source_url = https://github.com/tree-sitter/tree-sitter/archive/refs/tags/0.15.7.tar.gz
+source_filename = 0.15.7.tar.gz
```

Comment:
This local save-to-filename should include "tree-sitter" at the beginning, rather than just containing the version.